### ISSUE TITLE
IP Not Found

### ISSUE THREAD

**Issue by KieronWiltshire** [kubernetes/minikube#4311]
I’m running minikube on windows 10, attempting to use HyperV driver.

I created an external network switch and everything, but it failed to find the IP.

The message returned is “IP Not Found”

I’ve even decided to run this on a clean install of windows 10 latest update.

No avail.

I’ve searched for the answers for days now, multiple reinstalls of the OS. I believe this issue has occurred before in the past, but I’m yet to find a fix as most of those issues are now closed.

Sorry for vague description, 

I’m using minikube latest, kubectl latest, windows 10 pro latest etc...

Nothing seems to work!
Any help is greatly appreciated!

**Comment by tstromberg** [comment]
Thank you for sharing your experience! If you don't mind, could you please provide:

*  The exact command-lines used, so that we may replicate the issue 
*  The full output of the command that failed

I suspect that this may be a duplicate of #3648 but would like to confirm before closing this.  These commands will help us isolate the problem further. Thank you!

**Comment by KieronWiltshire** [comment]
@tstromberg it's definitely a duplicate, and I'm unable to post the full output as I've reinstalled windows etc... but after some digging, it looks to be an issue within windows.

You can't install a version less than 1809 and upgrade to get this to work, it has to be a clean install of 1809, I used 1903, now it all works correctly, well other than it now failing to pull the pods, but that's an issue for another time.

**Comment by uraniumsteel** [comment]
Got this to work by:
In Hyper-V manager creating new Switch with External turned on
After minikube created VM then failed by not finding IP, using Hyper-V manager changed the switch to the new switch I created with external.  Stopped the minikube vm and then restarted it, then ran minikube start again.     Minikube was then able to talk to the VM.   Just adding the switch did not work until I stopped and restarted the minikube vm in hyper-v manager.
